library/ssl_tls.c

```diff
@@ -6413,11 +6494,23 @@ static uint16_t ssl_preset_default_sig_algs[] = {
     MBEDTLS_TLS1_3_SIG_RSA_PSS_RSAE_SHA256,
 #endif
     MBEDTLS_TLS1_3_SIG_RSA_PKCS1_SHA256,
-
+#endif /* MBEDTLS_SSL_PROTO_TLS1_3 */
+#if defined(MBEDTLS_SSL_PROTO_TLS1_2)
```

Comment:
I did not think about that before but if both MBEDTLS_SSL_PROTO_TLS1_3 and MBEDTLS_SSL_PROTO_TLS1_2 are defined we may have duplicates here and we should avoid.